doc/users/customizing.rst

```diff
@@ -131,7 +135,7 @@ locations, in the following order:
 1. :file:`matplotlibrc` in the current working directory, usually used for
    specific customizations that you do not want to apply elsewhere.
 
-2. :file:`$MATPLOTLIBRC/matplotlibrc`.
```

Comment:
I do not see how it grabs a file from `$MATPLOTLIBRC` (instead of being a directory) from looking at the source....